client/src/components/AllEntries.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react'
+
+export default function AllEntries() {
+    fetch("/allEntries")
+    .then((res) => res.json())
+    .then((entry) => {
+      entry.forEach((element) => {
+        // container
+        let Cont = document.createElement("div");
+        Cont.id = element._id;
+        Cont.className = "listing-cont";
+
+        //title element
+        let Title = document.createElement("h3");
+        Title.innerText = element.title;
+        Title.className = "listing-title";
+
+        //author element
+        let Author = document.createElement("h5");
+        Author.innerText = element.author;
+        Author.className = "listing-author";
+
+        //text area element
+        let TextArea = document.createElement("h5");
+        TextArea.innerText = element.textArea;
+        TextArea.className = "listing-textArea";
+
+        //Category element
+        let Category = document.createElement("h5");
+        Category.innerText = element.category;
+        Category.className = "listing-category " + element.category;
+
+        //Date element
+        let Dates = document.createElement("h5");
+        Dates.innerText = element.date;
+        Dates.className = "listing-date";
+
+        //Edit button
+        let Edit = document.createElement("button");
+        Edit.innerText = "Edit";
+        Edit.className = "listing-edit";
+        Edit.id = element._id;
+        Edit.onclick = Editing;
+
+        //Delete button
+        let Delete = document.createElement("button");
+        Delete.innerText = "Delete";
+        Delete.className = "listing-delete";
+        Delete.onclick = Deleting;
+
+        //adding all elements to the container
+        Cont.appendChild(Edit);
+        Cont.appendChild(Delete);
+        Cont.appendChild(Title);
+        Cont.appendChild(Author);
+        Cont.appendChild(TextArea);
+        Cont.appendChild(Category);
+        Cont.appendChild(Dates);
+        //adding the container to the page
+        document.getElementById("full-list").appendChild(Cont);
```

Comment:
rather than doing a `document.appendChild` at this point, we could have just set the whole JSX to a variable. and in your code at the bottom you can just place that variable into your `<div>` that exists in your JSX already